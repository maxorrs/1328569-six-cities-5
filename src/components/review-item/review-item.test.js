import React from 'react';
import renderer from 'react-test-renderer';

import ReviewItem from './review-item';

const reviewMock = {
  user: {
    name: `A`,
    avatarUrl: `1`
  },
  comment: `123`,
  rating: 2,
  date: `2020-10-28T11:53:40.137Z`
};

it(`ReviewItem is rendered correctly`, () => {
  const tree = renderer
    .create(
        <ReviewItem
          review={reviewMock} />
    )
  .toJSON();

  expect(tree).toMatchSnapshot();
});
