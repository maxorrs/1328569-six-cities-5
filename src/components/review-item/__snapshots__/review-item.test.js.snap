// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewItem is rendered correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="1"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      A
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "40%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      123
    </p>
    <time
      className="reviews__time"
      dateTime="2020-10-28"
    >
      October 2020
    </time>
  </div>
</li>
`;
